src/pages/Viewer/PageSlider.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import theme from '../../styles/theme';
+
+const PageSlider = ({ currentPage, setCurrentPage, numPages }) => {
+  const [slidePage, setSlidePage] = useState(currentPage);
+
+  const handleOnchange = e => {
```

Comment:
```suggestion
  const handleSlidePage = e => {
```
구현 방식(how)보다는 대상에 대하여(what) 이름을 짓는 것이 좋습니다